README.md

```diff
@@ -66,6 +66,34 @@ pip install -r requirements.txt
 
 This will install the required Python packages in a virtual environment.
 
+## Resource Group
+
+This module requires an existing Azure Resource Group. You can either:
+
+1. Create one with Terraform before running this module:
+
+    ```hcl
+    resource "azurerm_resource_group" "materialize" {
+      name     = var.resource_group_name
+      location = var.location
+    }
+    ```
+
+    Then set the `resource_group_name` variable in your `terraform.tfvars` file:
+
+    ```hcl
+    resource_group_name = "your-desired-rg-name"
+    ```
+
+2. Use an existing one by just setting the name in your `terraform.tfvars` file:
+
+    ```hcl
+    resource_group_name = "your-existing-rg"
+    ```
+
+> [!IMPORTANT]
+> This module does not create a resource group. You must create one separately and set the `resource_group_name` variable explicitly. Setting it in your `terraform.tfvars` file is recommended to avoid accidental overwrites or mismatches.
```

Comment:
Small question  ... if they declare the resource in this main.tf, doesn't it create the resource group? At least, it did for me last week.  I'll recheck.

Reply:
Works for me!  Thank you!